README.md

```diff
@@ -231,6 +231,20 @@ status = "generate"
 Note that you must not place `Gtk.*` into the `generate` array and
 additionally configure its members.
 
+You can control the generation of constants in a similar fashion:
+
+```
+[[object]]
+name = "Gtk.*"
+status = "generate"
+    [[object.constant]]
+    pattern = "*"
```

Comment:
Just not to forget about problem:
I missed that wrong pattern format used: it not glob pattern but regex, so better use ".+" here.
It works because actual pattern looks "^*$"

Reply:
hrm, then I'm not sure how it even worked at all :)